docs/UserGuide.md

```diff
@@ -799,6 +874,8 @@ allow users to add more types of events in this application.
 * Add performance `performance add`
 * View performance list `performance list`
```

Comment:
I think this command is changed to performance `list` right? Perhaps can update here also.

Reply:
right!